lte/cloud/go/services/subscriberdb/load.go

```diff
@@ -231,3 +231,37 @@ func ConvertSubEntsToProtos(ent configurator.NetworkEntity, apnConfigs map[strin
 
 	return subData, nil
 }
+
+func LoadSuciProtosPage(
+	ctx context.Context, networkID string) ([]*lte_protos.SuciProfile, error) {
+
+	network, _ := configurator.LoadNetwork(ctx, networkID, true, true, serdes.Network)
+
+	ngcModel := &lte_models.NetworkNgcConfigs{}
+	ret := ngcModel.GetFromNetwork(network)
+	if ret == nil {
+		return nil, nil
+	}
+
+	suciProfileList := ret.(*lte_models.NetworkNgcConfigs).SuciProfiles
+	suciProtos := []*lte_protos.SuciProfile{}
+	for _, suciProfile := range suciProfileList {
```

Comment:
General nit: instead of `fooList`, just make it plural -- `foos`

Reply:
ok